shared/go/pkg/server/api_spa_server_test.go

```diff
@@ -0,0 +1,181 @@
+package server
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type APIRes struct {
+	apiRoute     string
+	requestRoute string
+	apiResponse  string
+	expected     string
+	shouldPass   bool
+}
+
+var apiTestcases = []APIRes{
+	{"/api/v1/status", "/api/v1/status", "OK", "OK", true},
+	{"/api/v1/status", "/api/v1/status", "OK", "Ok", false},
```

Comment:
If the input is the same, why should a test first pass and then not pass?

Reply:
ah, the `OK` is case sensitive.

Reply:
exactly, lower case... I could have gone for something more obvious, I guess